app/assets/javascripts/application.js

```diff
@@ -229,28 +229,32 @@ function filter_by_level(item){
 
   // Note that class names don't map to log level names (label-info == notice)
   if(level == 'info'){
-    $('.label-info').closest('tr').show();
-    $('.label-default').closest('tr').show();
-    $('.label-warning').closest('tr').show();
-    $('.label-danger').closest('tr').show();
+    $('.label-info.result-filter-tag').closest('tr').show();
+    $('.label-default.result-filter-tag').closest('tr').show();
```

Comment:
Just to provide a background where this example comes from, we hit this in openscap plugin where we have such table and we'd like to use the existing JS function from core. So we need to identify the column (result-filter-tag)